include/openssl/e_os2.h

```diff
@@ -206,8 +206,12 @@ extern "C" {
 #  endif
 # endif
 
-# if defined(OPENSSL_SYS_UEFI) && !defined(ssize_t)
-#  define ossl_ssize_t int
+# if defined(OPENSSL_SYS_UEFI) && !defined(ossl_ssize_t)
+#  if !defined(ssize_t)
+#   define ossl_ssize_t int
+#  else
+#   define ossl_ssize_t ssize_t
+#  endif
 #  define OSSL_SSIZE_MAX INT_MAX
```

Comment:
@dot-asm. 
Thanks. I agree. Any piece of code should be readable by itself and consistent with itself.
I updated the patch to use UEFI-aware data type (INTN) directly for this definition refining, which is processor-binding type. (I just realized we already added some "typedef" in e_os2.h for UEFI build, so it's better to leverage the similiar UEFI type directly under OPENSSL_SYS_UEFI scope).